### ISSUE TITLE
brew rm $(brew deps pygtk) fail

### ISSUE THREAD

**Issue by davidak** [Homebrew/homebrew#39903]
i am following this guide: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Common-Issues.md#python-segmentation-fault-11-on-import-some_python_module

```
brew rm $(brew deps pygtk)
Error: No such keg: /usr/local/Cellar/git
```

my git is not installed via brew.
maybe this command should ignore not installed packages.

i had to delete some packages from the list:

```
brew rm atk cairo fontconfig freetype gdk-pixbuf gettext glib gobject-introspection gtk+ harfbuzz icu4c jpeg libffi libpng libtiff pango pixman pkg-config py2cairo pygobject python
```

**Comment by DomT4** [comment]
Should work if you do something like ```brew rm `brew deps pygtk --installed` ```

**Comment by davidak** [comment]
that shows all dependencies of all installed packages. `pygtk`has no effect

```
$ brew deps --installed
atk: glib gobject-introspection pkg-config xz
brew-cask: git ruby
cairo: fontconfig freetype glib libpng pixman pkg-config xz
fontconfig: freetype pkg-config
freetype: libpng
gdbm:
gdk-pixbuf: glib gobject-introspection jpeg libpng libtiff pkg-config xz
gettext: xz
glib: gettext libffi pkg-config xz
gobject-introspection: git glib libffi pkg-config xz
gtk+: atk gdk-pixbuf gobject-introspection jasper pango pkg-config xz
harfbuzz: cairo freetype glib icu4c pkg-config
htop-osx: autoconf automake libtool
icu4c:
jpeg:
libffi:
libpng: xz
libtiff: jpeg
openssl: makedepend
pango: cairo fontconfig glib gobject-introspection harfbuzz pkg-config xz
pixman: pkg-config
pkg-config:
py2cairo: cairo pkg-config
pygobject: glib pkg-config python
pygtk: atk glib gtk+ libglade pkg-config py2cairo pygobject
python: gdbm homebrew/dupes/tcl-tk openssl pkg-config readline sqlite
readline:
sqlite: icu4c readline
vala: gettext glib pkg-config xz
```

**Comment by DomT4** [comment]
Fair point. It's an ugly workaround, and I'm surprised we don't support listing only the installed deps, but `brew deps --installed | grep -e pygtk: | /usr/bin/awk '{sub(/pygtk: /, ""); print $0}' | xargs brew remove` would work.

**Comment by xu-cheng** [comment]
Fixed by #40233.

Now you should be able to do `brew rm $(brew deps --installed pygtk)`

**Comment by davidak** [comment]
thank you very much!
